src/components/layer-metadata/layer-metadata.vue

```diff
@@ -54,145 +55,147 @@ function closeLayerMetadata() {
+    :backdrop="false"
+    :footer="false"
+    :title="t(`${layerMetadata.title}`, { ns: 'client' })"
+    @close="closeLayerMetadata"
   >
-    <div class="lux-modal-header flex flex-row justify-between">
-      <h1>{{ t(`${layerMetadata.title}`) }}</h1>
-      <button @click="closeLayerMetadata">X</button>
-    </div>
-    <div class="grid gap-2 grid-cols-3 pt-3 text-[13px] font-arial break-words">
-      <layer-metadata-item
-        v-if="layerMetadata.name"
-        :label="t('Name')"
-        :value="layerMetadata.name"
-      ></layer-metadata-item>
-      <layer-metadata-item
-        v-if="layerMetadata.serviceDescription"
-        :label="t('Description du Service')"
-        :value="layerMetadata.serviceDescription"
-      ></layer-metadata-item>
+    <template v-slot:content>
       <div
-        v-if="layerMetadata.description"
-        class="col-span-3 grid gap-2 grid-cols-3"
+        class="grid gap-2 grid-cols-3 pt-3 text-[13px] font-arial break-words"
       >
-        <span class="font-bold">{{ t('Description') }}</span>
-        <span class="col-span-2">
-          <span v-dompurify-html="description"></span>
-          <button
-            :title="
-              t('Display full description', {
-                ns: 'client',
-              })
-            "
-            v-if="!displayFullDescription"
-            @click="showFullDescription"
-            class="text-secondary hover:underline"
-          >
-            ...
-          </button>
-          <button
-            :title="
-              t('Hide full description', {
-                ns: 'client',
-              })
-            "
-            v-if="
-              displayFullDescription &&
-              (description?.length || '') > MAX_DESCRIPTION_LENGTH
-            "
-            @click="hideFullDescription"
-            class="text-secondary hover:underline"
-          >
-            -
-          </button>
-        </span>
-      </div>
-      <layer-metadata-item
-        v-if="layerMetadata.legalConstraints"
-        :label="t(`Contrainte d'utilisation`)"
-        :value="layerMetadata.legalConstraints"
-      ></layer-metadata-item>
-      <div class="col-span-3" v-if="layerMetadata.link?.length !== 0">
+        <layer-metadata-item
+          v-if="layerMetadata.name"
+          :label="t('Name')"
+          :value="layerMetadata.name"
+        ></layer-metadata-item>
+        <layer-metadata-item
+          v-if="layerMetadata.serviceDescription"
+          :label="t('Description du Service')"
+          :value="layerMetadata.serviceDescription"
+        ></layer-metadata-item>
         <div
-          class="grid gap-2 grid-cols-3"
-          v-for="link in layerMetadata.link"
-          :key="link"
+          v-if="layerMetadata.description"
+          class="col-span-3 grid gap-2 grid-cols-3"
         >
-          <span class="font-bold">{{ t('Url vers la resource') }}</span>
+          <span class="font-bold">{{ t('Description') }}</span>
```

Comment:
This is because the `layer-metadata.vue` was developed in the very beginning. As we've added `{ ns: 'client' }` later on, there is a mix here now. Will keep as is for now.